embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/session/message/PayloadFactoryImpl.kt

```diff
@@ -17,7 +17,7 @@ internal class PayloadFactoryImpl(
 
     private val collator: PayloadMessageCollator
         get() = when {
-            configService.sessionBehavior.useV2Payload() -> v2payloadMessageCollator
+            configService.oTelBehavior.isStableEnabled() -> v2payloadMessageCollator
```

Comment:
This should be dev - this is the least stable right now and will require us migration everything previously in SessionMessage to be able to turn on 😅

Reply:
Updated